### DISCUSSION TITLE
I cant install

### DISCUSSION THREAD

**Discussion by KrOOb-5** [CrankshaftForSteam/crankshaft#29] [General]
Hey, i followed the steps.

Install and then launch, there’s the icon on the right but i cant find the plugins tab or the puzzle icon in the desktop mode :(

**Comment by DrakiaXYZ** [comment]
Hey KrOOb!

Sorry for the delay in getting a fix out for this. We currently do not have access to pushing releases, so we had to come up with a temporary solution that will get users back on their feet, while we work on getting the ability to publish releases.

If you join the Crankshaft Discord, there is a pinned message in the #crankshaft-support channel with instructions on applying the patch to get everything going in the new SteamOS release.

Crankshaft Discord: https://discord.gg/bjEHdmvwEw
